### ISSUE TITLE
Pull Replication can't handle compressed attachment 

### ISSUE THREAD

**Issue by huytd** [couchbase/couchbase-lite-ios#837]
I'm working on an iOS app that use CouchbaseLite to do a pull replication to fetch the data from remote CouchDB server.

When I trying to do the pull, I got this error:
```
15:56:44.546‖ WARNING: Oops, server is trying to send a compressed attachment, I can't do that yet
15:56:44.546‖ WARNING: CBLMultipartDocumentReader[_id="_design/pages"]: received unparseable MIME multipart response: Stopped
```

Sound like Couchbase Lite iOS can't handle the compressed attachments.

We also have an Android app that use Couchbase Lite Android, but it can fetch the data normally without any problem.

Did anyone got this problem before? And do you know any solution to fix it?

Thanks

**Comment by pasin** [comment]
Yes, here is the issue #786 that was reported before. The issue was fixed in master branch and will be included in 1.1.1 release. Meanwhile, could you try to build the binary from the master branch yourself? Here is the instruction: https://github.com/couchbase/couchbase-lite-ios/wiki/Building-Couchbase-Lite.

**Comment by tiulkin** [comment]
Still does't work to me. Actually, I can't figure out the difference between attachments that replicates and attachments that don't. All they are small pngs, but 30 out of 200 won't replicate.
